man/nanotime.Rd

```diff
@@ -85,6 +88,9 @@ A nanotime object
 Functions to operate on nanosecond time resolution using integer64
 bit representation. Convertions functions for several standard R
```

Comment:
also, further down:

'64 bit integers whereas R itself _onky_ has 32 bit integers' ->
'64-bit integers whereas R itself _only_ has 32-bit integers'

don't know if you want 32- and 64-bit hyphenated.  you use both in this document